addons/api/mirage/serializers/policy.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  modelName: 'policy',
```

Comment:
Even though we don't use a serializer for policies, I was getting `a mirage model not registered error` from the mirage, so I had to add this file. let me know if there's another to skip this

Reply:
I think we still need it for mirage but I haven't dug into that